src/utils/certify.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+var certifiedTokens = {
```

Comment:
the `certifiedTokens` should have two list. One for Testnet and another for Mainet. The return list will depend on Network Byte.